### PR TITLE
View other profiles

### PR DESCRIPTION
**Pull request by kaykakaraka**
links to other user's profile by clicking on their email address

### CODE REVIEW

File: controllers/users.js [hunk 1]
```diff
@@ -17,18 +17,30 @@ const UsersController = {
...
-    
+    console.log(req.session);
+    console.log(req.body);
+    Post.find({user: req.body.email}, (err, posts) => {
+      if (err) {
+        throw err;
+      } else {
+      posts.reverse() // reorders posts, so newest post is always at the top of the list
+      console.log(posts);
+      res.render("users/profile", {posts: posts, email: req.body.email, session: req.session});
```
Comment (michaelcychan) [inline controllers/users.js:41]: Nice way to reuse the same profile.hbs! Great job!

File: cypress/integration/user_can_view_other_profile_pages_spec.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+describe("User can view others' Profile Page", function() {
```
Comment (michaelcychan) [inline cypress/integration/user_can_view_other_profile_pages_spec.js:1]: Good tests!

Comment (Hafizalimi7) [inline cypress/integration/user_can_view_other_profile_pages_spec.js:1]: very clean and understandable

File: public/stylesheets/style.css [hunk 3]
```diff
@@ -81,4 +81,9 @@ li.each-message {
   list-style-type: none;
   background-color: rgb(196, 225, 229);
   display:table
+}
```
Comment (Hafizalimi7) [inline public/stylesheets/style.css:84]: nice css

**Review by Hafizalimi7 — APPROVED** [review]

File: views/layout.hbs [hunk 4]
```diff
@@ -23,7 +23,7 @@
       {{!-- removed home button--}}
       {{#if session.user.email }}
         <input type="button" class="navigation-button" id="view-self-profile" value="Profile Page" onclick="location.href='/users/profile'"></input>
-        <input type="button" class="navigation-button" id="view-posts" value="📜 Posts" onclick="location.href='/posts/'"></input>
+        <input type="button" class="navigation-button" id="view-posts" value="📜 Posts" onclick="location.href='/posts'"></input>
```
Comment (michaelcychan) [inline views/layout.hbs:26]: interesting /

**Pull request merged**